onnxruntime/core/graph/function_utils.cc

```diff
@@ -373,7 +373,8 @@ class Inliner {
   // Replace given name with a unique version of the name, and cache the
   // renaming-binding in current scope.
   void make_unique(std::string& name) {
-    auto new_name = prefix_ + name;
+    auto new_name{prefix_};
+    new_name.append("/").append(name);
```

Comment:
I think it would be better to stick to ONNX's requirements for identifiers (several exporters violate it, and it is a hassle in some contexts). That would mean not using "/" in the identifier ... "_" should be ok. This will help if the model is exported to ONNX format subsequent to this renaming.